README.md

```diff
@@ -8,6 +8,7 @@
 [![Project Chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg?style=for-the-badge&logo=zulip)](https://quarkusio.zulipchat.com/)
 [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?style=for-the-badge&logo=gitpod&logoColor=white)](https://gitpod.io/#https://github.com/quarkusio/quarkus/-/tree/main/)
 [![Supported JVM Versions](https://img.shields.io/badge/JVM-11--17--19-brightgreen.svg?style=for-the-badge&logo=openjdk)](https://github.com/quarkusio/quarkus/actions/runs/113853915/)
+[![Gradle Enterprise](https://img.shields.io/badge/gradle_enterprise-build_scans-brightgreen.svg?style=for-the-badge&logo=gradle)](https://ge.quarkus.io/scans)
```

Comment:
applied without the color as thats the main thing that makes it look "out of place".

I personal prefer it says what it does rather than "revved up by" but if thats what you have everywhere else I'm fine with that.